src/index.html

```diff
@@ -1,13 +1,56 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <div>
+        <img
+          src="./images/imac.jpeg"
+          alt="mac"
+          class="card__img">
+      </div>
+
+      <div class="card__text">
+        <span class="card__text--description">Моноблок APPLE A1419 iMac 27&#8243; Retina 5K (MNED2UA/A)</span>
```

Comment:
```suggestion
      <div class="card__text card__text--description">
        <span class="">Моноблок APPLE A1419 iMac 27&#8243; Retina 5K (MNED2UA/A)</span>
```
блок__элемент блок__элемент--модификатор